modules/calc/src/main/java/com/opengamma/strata/calc/runner/function/CalculationFunction.java

```diff
@@ -83,6 +86,20 @@
   public abstract Currency naturalCurrency(T target, ReferenceData refData);
 
   /**
+   * Manages the calculation tasks.
+   * <p>
+   * This allows a function to control the tasks that are executed to calculate the results.
+   * <p>
+   * The tasks are created using a dedicated builder that provides access to the target and columns.
+   * Customization of the tasks can be achieved by calling methods on the builder.
+   * 
+   * @param builder  the calculation task builder
+   */
+  public default void manageTasks(TaskBuilder builder) {
```

Comment:
Was called `createTasks()` and returned the tasks. But this leads to potentially unsafe states where not all cells have an associated result calculation.